packages/frontend/src/components/DashboardTiles/TileBase/TileBase.styles.tsx

```diff
@@ -20,7 +20,7 @@ export const TileBaseWrapper = styled.div<HeaderContainerProps>`
     ${(props) =>
         props.$isEditMode
             ? `border: 1px dashed #7ea5ff;`
-            : `box-shadow: 0 0 0 1px #11141826;`}
+            : `box-shadow: 0 0 0 1px #bec1c426;`}
```

Comment:
@joaoviana we should do the same with this one, we need to leave FIXME on hardcoded values to replace them once we have a proper theme config